packages/perseus/src/widgets/explanation/explanation.testdata.ts

```diff
@@ -88,15 +61,14 @@ export const ipsumExample: PerseusRenderer = {
                     I haven't determined if our phaser energy can generate a stable field.
                     We could alter the photons with phase discriminators.
                 `,
-                static: false,
                 showPrompt: "Explanation",
-            },
-            alignment: "default",
-        },
+                widgets: {},
```

Comment:
Is this something that is included in the default options?

Reply:
Yes! This is the default options object inside Explanation's widgetLogic:

```
const defaultWidgetOptions: ExplanationDefaultWidgetOptions = {
    showPrompt: "Explain",
    hidePrompt: "Hide explanation",
    explanation: "explanation goes here\n\nmore explanation",
    widgets: {},
};
```

Reply:
OH I see. I can remove it.